src/main/scala/com/raphtory/core/deploy/RaphtoryService.scala

```diff
@@ -9,6 +9,46 @@ import org.slf4j.LoggerFactory
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe.TypeTag
 
+/**
+  * {s}`RaphtoryService[T]`
+  *
+  *  : `RaphtoryService` for initialising spout, graphbuilder, deploying spout, partition and query manager
```

Comment:
this is actually an abstract class, that has its some of its functions implemented later on within the code base, this doc should talk about this, what does this service do? examples of where it is used?  alot of our examples extend this class and add functionality, its one of the core components, the current description does not mention this